appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
Actually it seems to me that, that blocks like this are called "generic method blocks" (https://github.com/mit-cml/appinventor-sources/blob/master/appinventor/blocklyeditor/src/drawer.js#L221)

![screen shot 2015-04-12 at 12 26 24 pm](https://cloud.githubusercontent.com/assets/4552296/7106443/06ffdae8-e10f-11e4-9b8a-ca8fe932d176.png)

Which makes me feel like it makes sense to call these "generic event blocks"

![screen shot 2015-04-04 at 2 14 13 pm 2](https://cloud.githubusercontent.com/assets/4552296/7106445/1d78b25e-e10f-11e4-8f25-d01de90ebe9c.png)

But since these are inherently different blocks, I call these "multi-generic event blocks" and/or "multi event blocks"

![screen shot 2015-04-06 at 7 49 26 pm](https://cloud.githubusercontent.com/assets/4552296/7106448/57629872-e10f-11e4-930e-e9099e634775.png)

I don't think it make sense to call those generic events as well, but another name is certainly possible